openfisca_france/parameters/covid19.yaml

```
@@ -38,13 +38,33 @@ aide_exceptionnelle_famille:
         value: 100
       2020-11-01:
         value: null
-indemnite_ap :
-   taux : 
+indemnite_ap:
+   taux:
      2020-03-01:
-       0.70   
-   plancher :
+       0.70
+   plancher:
      2020-03-01:
        8.03
-   plafond_smic :
-     2020-03-01: 
+   plafond_smic:
+     2020-03-01:
        4.5
+aide_jeunes_diplomes_anciens_boursiers:
+  reference: https://www.pole-emploi.fr/candidat/mes-droits-aux-aides-et-allocati/allocations-et-aides--les-repons/aides-financieres-aux-jeunes-dip.html
+  age_limite:
+    description: Âge maximal limite exclu d'éligibilité à l'aide jeunes diplômes anciens boursiers
+    unit: year
+    values:
+      2021-02-05:
+        value: 30
+  taux_bourse:
+    description: Pourcentage du montant net de la bourse perçue la dernière année et reversé par l'aide
+    unit: rate
```

```suggestion
    unit: /1
```

[Source](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html).